Update cloud 24.07 RN (#3545)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-cloud/current/releases/cloud-release-notes.md

### Centreon IT Edition

> **SNMP community**

Reviewer: ```suggestion
**Security**
```

> As we keep improving the security of our software, the SNMP community is now considered as a password. As such, it is now obfuscated in host and host template forms. There is no way to retrieve the SNMP community from Centreon, and Centreon cannot check it for you. Users can still edit the SNMP community. Please note that exporting the monitoring configuration is required to enforce changes.

Reviewer: ```suggestion
As we keep improving the security of our software, the SNMP community is now considered as a password. As such, it is now obfuscated in host and host template forms. There is no way to retrieve the SNMP community from Centreon, and Centreon cannot check if its value is correct. Users can still edit the SNMP community. Please note that exporting the monitoring configuration is required to enforce changes.
```
